src/Subsampling/include/gudhi/choose_n_farthest_points.h

```diff
@@ -88,34 +88,57 @@ void choose_n_farthest_points(Distance dist,
     starting_point = dis(gen);
   }
 
-  std::size_t current_number_of_landmarks = 0;  // counter for landmarks
-  static_assert(std::numeric_limits<double>::has_infinity, "the number type needs to support infinity()");
   // FIXME: don't hard-code the type as double. For Epeck_d, we also want to handle types that do not have an infinity.
```

Comment:
Ok regarding #413 
Maybe we should define a type instead of using double to identify the future fix.

Reply:
I think that can be done in the future PR that will fix #413. I believe I made such a fix much simpler, but I'd rather put that in a separate patch.